static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js

```diff
@@ -1510,4 +1510,171 @@ describe('AccessMethod component', () => {
       })
     })
   })
+
+  describe('when the selected access method is swodlr', () => {
+    test('SWODLR Option displayed', async () => {
+      const collectionId = 'collectionId'
+      setup({
+        accessMethods: {
+          swodlr: {
+            type: 'SWODLR',
+            supportsSwodlr: true
+          }
+        },
+        metadata: {
+          conceptId: collectionId
+        },
+        selectedAccessMethod: 'swodlr'
+      })
+
+      const swodlrText = screen.getByText('Generate with SWODLR')
+      expect(swodlrText).toBeInTheDocument()
+    })
+
+    test('selecting a granuleExtent calls onUpdateAccessMethod', async () => {
+      const user = userEvent.setup()
+
+      const collectionId = 'collectionId'
+      const { onUpdateAccessMethod } = setup({
+        accessMethods: {
+          swodlr: {
+            type: 'SWODLR',
+            supportsSwodlr: true
+          }
+        },
+        metadata: {
+          conceptId: collectionId
+        },
+        selectedAccessMethod: 'swodlr'
+      })
+
+      const granuleExtent256Checkbox = screen.getByRole('radio', { name: '256 x 128 km' })
+      await user.click(granuleExtent256Checkbox)
+
+      expect(onUpdateAccessMethod).toHaveBeenCalledTimes(1)
+      expect(onUpdateAccessMethod).toHaveBeenCalledWith({
+        collectionId: 'collectionId',
+        method: {
+          swodlr: {
+            swodlrData: {
+              params: {
+                rasterResolution: 90,
+                outputSamplingGridType: 'UTM',
+                outputGranuleExtentFlag: true
+              },
+              custom_params: {}
+            }
+          }
+        }
+      })
+    })
+
+    test('selecting a LAT/LON sample grid calls onUpdateAccessMethod with automatic rasterResolution value adjustment', async () => {
+      const user = userEvent.setup()
+
+      const collectionId = 'collectionId'
+      const { onUpdateAccessMethod } = setup({
+        accessMethods: {
+          swodlr: {
+            type: 'SWODLR',
+            supportsSwodlr: true
+          }
+        },
+        metadata: {
+          conceptId: collectionId
+        },
+        selectedAccessMethod: 'swodlr'
+      })
+
+      const latLonCheckbox = screen.getByRole('radio', { name: 'LAT/LON' })
+
+      await user.click(latLonCheckbox)
+
+      expect(onUpdateAccessMethod).toHaveBeenCalledTimes(1)
+      expect(onUpdateAccessMethod).toHaveBeenCalledWith({
+        collectionId: 'collectionId',
+        method: {
+          swodlr: {
+            swodlrData: {
+              params: {
+                rasterResolution: 3,
+                outputSamplingGridType: 'GEO',
+                outputGranuleExtentFlag: false
+              },
+              custom_params: {}
+            }
+          }
+        }
+      })
+    })
+
+    test.only('can update individual granules MGRS band and UTM Zone Adjust', async () => {
```

Comment:
```suggestion
    test('can update individual granules MGRS band and UTM Zone Adjust', async () => {
```
we gotta remove this one